plutus-contract/docs/contract-api.adoc

```diff
@@ -0,0 +1,195 @@
+[#contract-api]
+= Writing Plutus Contracts
+
+The `PlutusContract` type represents the client-side parts of Plutus contracts. `PlutusContract` values describe the interactions of contracts with the blockchain and with the user. They can be composed in different ways, allowing you to assemble contracts from the building blocks provided by the Plutus library and by other contract authors.
```

Comment:
Seems to just be `Contract` now? This probably needs the LHS treatment to keep this stuff current.

Reply:
It will replace the plutus tutorial eventually: https://github.com/input-output-hk/plutus/issues/1321